Modules/System/Time Zone Management/src/DateTimeOffset.Codeunit.al

```diff
@@ -0,0 +1,38 @@
+codeunit 50110 "DateTime Offset"
+{
+    /// <summary>
+    /// Retrieves the offset from the requested time zone at the time of the requested datetime. This takes into account any daylight saving time conditions that may apply.
+    /// </summary>
+    /// <param name="SourceDateTime">The datetime that will be used as the basis for the difference calculation.</param>
+    /// <param name="TimeZoneId">The ID of the time zone that you want to calculate the offset for.</param>
+    /// <returns>A duration that indicates the offset between UTC and the requested time zone for the provided datetime.</returns>
+    procedure GetUtcOffset(SourceDateTime: DateTime; TimeZoneId: Text): Duration
+    begin
+        exit(DateTimeConversionImpl.GetUtcOffset(SourceDateTime, TimeZoneId));
+    end;
+
+    /// <summary>
+    /// Retrieves the offset from the user time zone at the time of the requested datetime. This takes into account any daylight saving time conditions that may apply.
+    /// </summary>
+    /// <param name="SourceDateTime">The datetime that will be used as the basis for the difference calculation.</param>
+    /// <returns>A duration that indicates the offset between UTC and the user time zone for the provided datetime.</returns>
+    procedure GetUtcOffsetForUserTimeZone(SourceDateTime: DateTime; TimeZoneId: Text): Duration
```

Comment:
Signature of procedure should be updatet to only need the SourceDateTime

Reply:
Shows you should never code while on the phone. 

I fixed the signature and added some other minor fixes to the PR.